components/Dialogs/src/MDCAlertController.h

```diff
@@ -85,6 +85,25 @@
 + (nonnull instancetype)alertControllerWithTitle:(nullable NSString *)title
                                          message:(nullable NSString *)message;
 
+/**
+ Convenience constructor to create and return a view controller for displaying an alert to the user.
+
+ After creating the alert controller, add actions to the controller by calling -addAction.
+
+ @note Most alerts don't need titles. Use only for high-risk situations.
+
+ @note This method receives an @c NSAttributedString for the display message. Use
+       @c alertControllerWithTitle:message: for regular @c NSString support.
+
+ @note Currently, support for tappable embedded links within attributed strings is not supported.
```

Comment:
Please apply the same fix here as the one below.

Reply:
You got it